### ISSUE TITLE
Arxiv 26 | Spectral Progressive Diffusion for Efficient Image and Video Generation

### ISSUE THREAD

**Issue by NightDreamy** [AI-Infra-Team/awesome-papers#224]
Author: Xiao, Howard and Chao, Brian and Yariv, Lior and Wetzstein, Gordon
Paper: https://arxiv.org/abs/2605.18736
Github: https://github.com/howardhx/speed
 ## Motivation
### Spectral Autoregression in Diffusion Models

<img width="579" height="337" alt="Image" src="https://github.com/user-attachments/assets/6f540f56-be55-4391-bbcc-333b873110c3" />

 Ieverage this spectral autoregression property to progressively grow resolution within a single denoising trajectory via spectral domain transformations for efficient image and video generation.

### Limitation in current technical
Either require significant heuristics-based hyperparameter tuning or specialized modules.

###  Process
1.Compute the spectrum on the frequency set;
2.Embed 𝝃 in the lower-frequency part of the spectrum and fill the slots;
3.Convert the spectrum back to spatial domain.
### Timestep alignment.
the high-frequency noise padding raises the overall noise level of the enlarged:

<img width="200" height="121" alt="Image" src="https://github.com/user-attachments/assets/b3cebc6f-b97f-4390-8cfb-77efb9f85d4d" />

### Optimal Resolution Schedule
Spectral component ω is noise-dominated as its optimal velocity predictions are approximately noise. 

<img width="354" height="125" alt="Image" src="https://github.com/user-attachments/assets/d7bf8439-0174-4a74-9fac-f2361e53a4ca" />

It implies that high-resolution computation in early denoising steps is redundant as most representable frequencies ω are still noise-dominated.

### Finetuning
Construct the stage input by applying the same spectral noise expansion and timestep alignment used at inference, fixing training-inference gaps.

### Model
FLUX.1-dev+Z-image+Wan2.1-T2V-1.3B
### Result

<img width="1182" height="1075" alt="Image" src="https://github.com/user-attachments/assets/9afddbd6-4fbf-4091-8c4c-c56458ec15b2" />

<img width="580" height="495" alt="Image" src="https://github.com/user-attachments/assets/556bc16d-770e-4d4c-b726-1f81cea18459" />

<img width="576" height="181" alt="Image" src="https://github.com/user-attachments/assets/297f4455-33c5-44f5-b59e-7e8fba13874f" />
